README.md

```diff
@@ -4,6 +4,11 @@
 >- [Password stored in plaintext](https://jenkins.io/security/advisory/2018-07-30/#SECURITY-1039)
 >- [Credentials stored in plaintext](https://jenkins.io/security/advisory/2019-11-21/#SECURITY-1539)
 
+> ANCHORE ENGINE USERS NOTICE
```

Comment:
Not sure we need the full message here about scheduling demos etc. Maybe more just a message saying:
```As of 2023, [Anchore Engine](https://github.com/anchore/anchore-engine) is no longer maintained. Future versions of this Jenkins plugin _may_ break support for existing anchore-engine installations. No timeframe is scheduled. ```